### ISSUE TITLE
Argmin on 5D tensor failed with message "error while computing output extent" from graphShapeAnalyzer.cpp

### ISSUE THREAD

**Issue by lazycal** [onnx/onnx-tensorrt#803]
## Description

The `Argmin` op seems to fail on any 5D tensor, regardless of the shape. An example model:
![image](https://user-images.githubusercontent.com/7333325/152914398-6d932ee1-cfd1-4fc0-910d-b6c3bd0dd404.png)

```python
import numpy as np
import onnx
import onnx.checker
import torch

inputs = {
    'i3': np.random.rand(96, 1, 4, 1, 1).astype(np.float32),
}
class Model(torch.nn.Module):
    @torch.no_grad()
    def forward(self, i3):
        i3 = i3.argmin(dim=0)
        return i3


model = Model()
model.eval()
torch_inp = {k: torch.from_numpy(v) for k, v in inputs.items()}
torch.onnx.export(
    model, tuple(torch_inp.values()),
    "output.onnx", input_names=list(inputs.keys()),
    verbose=False, opset_version=13)

onnx_model = onnx.load("output.onnx")
onnx.checker.check_model(onnx_model, full_check=True)
```

Running `onnx2trt output.onnx`, I got
```
[2022-02-08 03:58:59   ERROR] [graphShapeAnalyzer.cpp::computeOutputExtents::1026] Error Code 4: Miscellaneous (ArgMin_0: error while computing output extent)
While parsing node number 0 [ArgMin -> "1"]:
ERROR: ModelImporter.cpp:162 In function parseGraph:
[6] Invalid Node - ArgMin_0
cannot create std::vector larger than max_size()
```

Did some digging, and it seems that the `addTopK` function is the culprit, because when I added `    LOG_VERBOSE("indices.dims=" << indices->getDimensions()) ;` right after L110:
https://github.com/onnx/onnx-tensorrt/blob/8b6144a88534e6fd15b6ef5457f1c1a1fc153939/onnx2trt_utils.cpp#L103-L110
it prints
```
[2022-02-08 03:50:09   ERROR] [graphShapeAnalyzer.cpp::computeOutputExtents::1026] Error Code 4: Miscellaneous (ArgMin_0: error while computing output extent)
[2022-02-08 03:50:09 UNKNOWN] indices.dims=()
```
where `indices.dims` should be `(1,4,1,1)` instead. This does not immediately crash the program, but later in this line https://github.com/onnx/onnx-tensorrt/blob/8b6144a88534e6fd15b6ef5457f1c1a1fc153939/onnx2trt_utils.cpp#L1929, the `dims.size()` is evaluated to -1, which caused the `std::vector` allocation failure.

## Environment

**TensorRT Version**: v8201
**ONNX-TensorRT Version / Branch**: main
**GPU Type**: RTX 2080
**Nvidia Driver Version**: 495.29.05
**CUDA Version**: 11.5
**CUDNN Version**: 8.3.0
**Operating System + Version**: Ubuntu 18.04
**Python Version (if applicable)**:  3.8.10
**TensorFlow + TF2ONNX Version (if applicable)**: 
**PyTorch Version (if applicable)**: 1.10.2+cpu
**Baremetal or Container (if container which image + tag)**: Container based on `nvcr.io/nvidia/tensorrt:21.11-py3`

## Relevant Files

[output.onnx.zip](https://github.com/onnx/onnx-tensorrt/files/8020681/output.onnx.zip)

## Steps To Reproduce
``onnx2trt output.onnx -v -v``, where output.onnx can be found in the attached.
Full trace of log:
```
----------------------------------------------------------------
Input filename:   output.onnx
ONNX IR version:  0.0.7
Opset version:    13
Producer name:    pytorch
Producer version: 1.10
Domain:           
Model version:    0
Doc string:       
----------------------------------------------------------------
[2022-02-08 04:11:10    INFO] [MemUsageChange] Init CUDA: CPU +322, GPU +0, now: CPU 333, GPU 1795 (MiB)
[2022-02-08 04:11:10    INFO] [MemUsageSnapshot] Begin constructing builder kernel library: CPU 333 MiB, GPU 1795 MiB
[2022-02-08 04:11:10    INFO] [MemUsageSnapshot] End constructing builder kernel library: CPU 467 MiB, GPU 1829 MiB
Parsing model
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::GridAnchor_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::GridAnchorRect_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::NMS_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::Reorg_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::Region_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::Clip_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::LReLU_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::PriorBox_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::Normalize_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::ScatterND version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::RPROI_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::BatchedNMS_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::BatchedNMSDynamic_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::FlattenConcat_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::CropAndResize version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::DetectionLayer_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::EfficientNMS_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::EfficientNMS_ONNX_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::EfficientNMS_TFTRT_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::Proposal version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::ProposalLayer_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::PyramidROIAlign_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::ResizeNearest_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::Split version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::SpecialSlice_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Registered plugin creator - ::InstanceNormalization_TRT version 1
[2022-02-08 04:11:10 UNKNOWN] Adding network input: i3 with dtype: float32, dimensions: (96, 1, 4, 1, 1)
[2022-02-08 04:11:10 UNKNOWN] Registering tensor: i3 for ONNX tensor: i3
[2022-02-08 04:11:10 UNKNOWN] Parsing node: ArgMin_0 [ArgMin]
[2022-02-08 04:11:10 UNKNOWN] Searching for input: i3
[2022-02-08 04:11:10 UNKNOWN] ArgMin_0 [ArgMin] inputs: [i3 -> (96, 1, 4, 1, 1)[FLOAT]], 
[2022-02-08 04:11:10 UNKNOWN] Registering layer: ArgMin_0 for ONNX node: ArgMin_0
[2022-02-08 04:11:10 WARNING] Tensor DataType is determined at build time for tensors not marked as input or output.
[2022-02-08 04:11:10   ERROR] [graphShapeAnalyzer.cpp::computeOutputExtents::1026] Error Code 4: Miscellaneous (ArgMin_0: error while computing output extent)
While parsing node number 0 [ArgMin -> "1"]:
--- Begin node ---
input: "i3"
output: "1"
name: "ArgMin_0"
op_type: "ArgMin"
attribute {
  name: "axis"
  i: 0
  type: INT
}
attribute {
  name: "keepdims"
  i: 0
  type: INT
}
attribute {
  name: "select_last_index"
  i: 0
  type: INT
}

--- End node ---
ERROR: ModelImporter.cpp:162 In function parseGraph:
[6] Invalid Node - ArgMin_0
cannot create std::vector larger than max_size()
```

**Comment by kevinch-nv** [comment]
Argmin/max is implemented via TensorRT's TopK layer, which has the limitaiton that the reduce axis must be in the last 4 dimensions of the tensor. This means that an argmax operating on axis 0 is unsupported for 5D tensors
